wherehows-web/app/templates/components/datasets/owners/suggested-owners.hbs

```diff
@@ -0,0 +1,35 @@
+<h4 class="dataset-authors-suggested__header">Suggestions</h4>
+
+{{#if isEmpty}}
+  <div class="dataset-authors-suggested__info__description">
+    We found no suggested owner(s) for this dataset, based on scanning different systems
+  </div>
+{{else}}
+  <section class="dataset-authors-suggested__info">
+    <div class="dataset-authors-suggested__info__facepile">
+      {{#each facepileOwners as |owner|}}
+        {{user-avatar class="dataset-authors-suggested__info__facepile__avatar" userName=owner.userName}}
+      {{/each}}
+    </div>
+    <div class="dataset-authors-suggested__info__description">
+      We found {{owners.length}} {{if (gt owners.length 1) "people" "person"}} who would be great owner(s)
+      for this dataset, based on scanning different systems
+    </div>
+    <div class="dataset-authors-suggested__info__trigger" {{action (mut isExpanded) (if isExpanded false true)}}>
```

Comment:
Can we make this a `button`, let's use semantic HTML components where applicable, and I think this would be better as a component action to toggleProperty, because we will start tracking  these user interactions in the short term

Reply:
Makes sense.